### PR TITLE
PHPUnit tests

### PR DESCRIPTION
**Pull request by YevhenShBolt**

### CODE REVIEW

File: Test/Unit/Block/JsTest.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+     * @var BlockJs
+     */
+    protected $block;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+//        $objectManager = new UnitObjectManager($this);
```
Comment (daisy1754) [inline Test/Unit/Block/JsTest.php]: let's remove commented out code

File: Test/Unit/Block/JsTest.php [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    }
+    /**
+     * Check if CDN Url equal or not.
+     *
+     * @param bool   $sandbox
+     * @param string $file
+     */
+    public function checkEquals($result, $sandbox = true, $file = '')
+    {
+        $mode = ($sandbox) ? HelperConfig::CDN_URL_SANDBOX : HelperConfig::CDN_URL_PRODUCTION;
```
Comment (daisy1754) [inline Test/Unit/Block/JsTest.php]: IMO test should not include condition. Can you rewrite test above without using this method, eg., 
> $this->assertEquals(HelperConfig::CDN_URL_SANDBOX . DIRECTORY_SEPARATOR . 'track.js', $result);

File: Test/Unit/Block/JsTest.php [hunk 3]
```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (YevhenShBolt) [reply Test/Unit/Block/JsTest.php]: done

File: Test/Unit/Block/JsTest.php (hunk 1 above)
Comment (YevhenShBolt) [reply Test/Unit/Block/JsTest.php]: done

File: Test/Unit/Block/JsTest.php (hunk 2 above)
Comment (YevhenShBolt) [reply Test/Unit/Block/JsTest.php]: done

File: Test/Unit/phpunit.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        </whitelist>
+    </filter>
+    <listeners>
+        <listener class="Magento\Framework\TestFramework\Unit\Listener\ReplaceObjectManager"/>
+    </listeners>
+    <logging>
+        <coverage_html_placeholder>
+            <log type="coverage-html" target="{{coverage_dir}}/test-reports/coverage" charset="UTF-8" yui="true" highlight="true"/>
+        </coverage_html_placeholder>
+        <!--coverage_cov_placeholder
```
Comment (daisy1754) [inline Test/Unit/phpunit.xml]: can you remove these lines?

File: Test/Unit/Block/JsTest.php [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    {
+        $file = 'connect.js';
+
+        // For CDN URL in sandbox mode
+        $this->setSandboxMode();
+        $result = $this->block->getConnectJsUrl();
+        $expectedUrl = HelperConfig::CDN_URL_SANDBOX . DIRECTORY_SEPARATOR . $file;
+        $this->assertEquals($result, $expectedUrl, 'Not equal CDN Url in Sandbox mode');
+
+        // ReInit for production mode.
```
Comment (daisy1754) [inline Test/Unit/Block/JsTest.php]: this test is testing two different things. can you split into two different methods?

File: Test/Unit/.gitignore [hunk 6]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (daisy1754) [inline Test/Unit/.gitignore]: please put these into top-level .gitignore

**Comment by daisy1754** [comment]
🎉

**Pull request merged**